src/rules/no_lazy_static.rs

```diff
@@ -0,0 +1,32 @@
+use crate::{
+    rule::{Rule, RuleBuilder, RuleListenerBuilder},
+    ViolationBuilder,
+};
+
+pub fn no_lazy_static_rule() -> Rule {
+    RuleBuilder::default()
```

Comment:
I could see having a generic rule-builder-helper for eg creating a rule that always reports all tree-sitter AST nodes that match a single tree-sitter query (with a message that is either static or a function of the tree-sitter AST node)

I was starting to picture something similar for easily generating "one-off" rules in (`tree-sitter-lint`-, not individual-rule-) tests eg `create_simple_replacer_rule("foo", "bar")`